Formula/mirror.rb

```diff
@@ -0,0 +1,24 @@
+class Mirror < Formula
+  version = "1.2"
+  desc "Command-line tool for fiddling with display mirroring: on/off/toggle"
+  homepage "https://fabiancanas.com/open-source/mirror-displays"
+  url "https://github.com/fcanas/mirror-displays/releases/download/v#{version}/mirror.zip"
+  sha256 "3a44b1e65fdbcd15ba93ec1a1af97e205c8d274cb0272a40940add9f62853e2f"
+  license "GPL-3.0-or-later"
+
+  resource "source" do
+    url "https://github.com/fcanas/mirror-displays/archive/refs/tags/v#{version}.tar.gz"
+    sha256 "96db4c12e8f6606db9822af54da35daaae0bff6853df7347cefdccd07a7e6acc"
+  end
```

Comment:
Why are we downloading what looks like a fork?

Reply:
Fair points on the rest of the comments, but I’m not sure why you think this is a fork. It’s coming from the same repository. It’s just that the binary release ZIP file doesn’t include the man page, so I thought it useful to get it from the source tarball. This won’t be an issue anyway when we compile from source.

Reply:
You're right, I had misread. Apologies!